Solutions/Menes.Specs/Features/OpenApiDefaultParameterParsing.feature

```diff
@@ -0,0 +1,91 @@
+﻿@perScenarioContainer
+
+Feature: OpenApiDefaultParameterParsing
+	In order to simplify setting values for parameters required for the underlying operation
+	As a developer
+	I want to be able to specify default values for those parameters within the OpenAPI specification and have those parameters validated and serialized for use downstream.
+
+Scenario Outline: Parameters with valid values for simple types
+	Given I have constructed the OpenAPI specification with a <ParameterLocation> parameter with name <ParameterName>, type <Type>, format <Format> and default value <DefaultValue>
+	When I try to parse the default value
+	Then the parameter <ParameterName> should be <ExpectedResult> of type <ExpectedResultType>
+
+	Examples:
+		| ParameterLocation | ParameterName   | Type    | Format    | DefaultValue                           | ExpectedResult                       | ExpectedResultType        |
+		| query             | openApiDate     | string  | date      | 2017-07-21                             | 2017-07-21T00:00:00Z                 | System.DateTimeOffset     |
+		| query             | openApiDateTime | string  | date-time | "2017-07-21T17:32:28Z"                 | 2017-07-21T17:32:28+00:00            | System.String             |
+		| query             | openApiPassword | string  | password  | myVErySeCurePAsSworD123                | myVErySeCurePAsSworD123              | System.String             |
+		| query             | openApiByte     | string  | byte      | U3dhZ2dlciByb2Nrcw==                   | U3dhZ2dlciByb2Nrcw==                 | ByteArrayFromBase64String |
+		| query             | openApiString   | string  | uuid      | 9b7d63fb-1689-4697-9571-00d10b873d78   | 9b7d63fb-1689-4697-9571-00d10b873d78 | System.Guid               |
+		| query             | openApiString   | string  | uri       | "https://myuri.com"                    | https://myuri.com                    | System.Uri                |
+		| query             | openApiString   | string  |           | "I said \"What is a 'PC'?\""           | I said "What is a 'PC'?"             | System.String             |
+		| query             | openApiBoolean  | boolean |           | true                                   | true                                 | System.Boolean            |
+		| query             | openApiLong     | integer | int64     | 9223372036854775807                    | 9223372036854775807                  | System.Int64              |
+		| query             | openApiInteger  | integer |           | 1234                                   | 1234                                 | System.Int32              |
+		| query             | openApiFloat    | number  | float     | 1234.5                                 | 1234.5                               | System.Single             |
+		| query             | openApiDouble   | number  | double    | 1234.5678                              | 1234.5678                            | System.Double             |
```

Comment:
As above - should we have a test for `number` without a specified schema format?

Reply:
Fixed.